### PR TITLE
CAM-14068: request platform information via the Camunda Webapps

### PR DESCRIPTION
**Pull request by cliftondavies**
related to CAM-14068

### CODE REVIEW

**Review by danielkelemen — APPROVED** [review]

File: admin/en.json [hunk 1]
```diff
@@ -188,6 +188,8 @@
     "TELEMETRY_SUCCESS": "Telemetry Configuration was saved successfully",
     "TELEMETRY_ERROR_STATUS": "Error",
     "TELEMETRY_ERROR_MESSAGE": "Your configuration could not be saved. Please try again.",
+    "DIAGNOSTICS": "Diagnostics",
+    "DIAGNOSTICS_INTRODUCTION": "To aid problem diagnosis by enabling customer facing teams to quickly understand the environment/distribution a user is running.",
```
Comment (tasso94) [inline admin/en.json:192]:
```suggestion
    "DIAGNOSTICS_INTRODUCTION": "The diagnostics data allows to understand the environment to aid problem diagnosis.",
```
This introduction sounds like a user story from a ticket description but might not make much sense when a user reads it.

File: admin/en.json [hunk 2]
```diff
@@ -548,6 +550,8 @@
     "PLGN_AUDIT_DELETE_NOTIFICATION_SUCCESS": "Annotation was deleted successfully.",
     "PLGN_AUDIT_DELETE_NOTIFICATION_FAILURE": "Annotation could not be deleted. ",
     "PLGN_AUDIT_MODAL_DELETE_CONFIRMATION": "Are you sure that you want to delete this annotation?",
+    "PLGN_MOD_REQUEST_PAYLOAD": "Copy platform data to clipboard",
+    "PLGN_MOD_SHOW_REQUEST_PAYLOAD": "View platform data",
```
Comment (tasso94) [inline admin/en.json:554]:
```suggestion
```
Do we need this translation string anymore?

**Review by tasso94 — CHANGES_REQUESTED** [review]

File: admin/en.json [hunk 3]
```diff
@@ -188,6 +188,10 @@
     "TELEMETRY_SUCCESS": "Telemetry Configuration was saved successfully",
     "TELEMETRY_ERROR_STATUS": "Error",
     "TELEMETRY_ERROR_MESSAGE": "Your configuration could not be saved. Please try again.",
+    "TELEMETRY_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed:",
```
Comment (tasso94) [inline admin/en.json:191]:
```suggestion
    "DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed: {{msg}}",
```

File: admin/en.json [hunk 4]
```diff
@@ -188,7 +188,7 @@
     "TELEMETRY_SUCCESS": "Telemetry Configuration was saved successfully",
     "TELEMETRY_ERROR_STATUS": "Error",
     "TELEMETRY_ERROR_MESSAGE": "Your configuration could not be saved. Please try again.",
-    "TELEMETRY_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed:",
+    "DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed: {{msg}}",
```
Comment (tasso94) [inline admin/en.json:191]: I changed the name of the token from `TELEMETRY_FETCH_DATA_ERROR_MESSAGE` to `DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE`. Also, I added a `msg` placeholder.

**Review by tasso94 — APPROVED** [review]
Looks good to me. Feel free to merge 🙂 👍

**Pull request merged**